src/Libraries/Microsoft.Extensions.AI.Abstractions/Utilities/AIJsonUtilities.Defaults.cs

```diff
@@ -116,6 +116,7 @@ private static JsonSerializerOptions CreateDefaultOptions()
     [JsonSerializable(typeof(Embedding<double>))]
     [JsonSerializable(typeof(AIContent))]
     [JsonSerializable(typeof(AIFunctionArguments))]
+    [JsonSerializable(typeof(ResponseContinuationToken))]
```

Comment:
This should be commented out for now. And once you do so, presumably you can again mark ResponseContinuationToken as [Experimental], revert the per-member annotations, and completely revert the json file?